packages/ember-metal/lib/mixin.js

```diff
@@ -257,8 +257,12 @@ function mergeMixins(mixins, m, descs, values, base, keys) {
 
 let IS_BINDING = /^.+Binding$/;
```

Comment:
Its publicly exported as `Ember.IS_BINDING`. I'm fine with killing that, but I don't know if folks are using it.  I guess I can add a deprecation in `ember-metal/index` for it when accessed.